### ISSUE TITLE
Is it possible to add Coderedcms to an existing wagtail website?

### ISSUE THREAD

**Issue by OBKF** [coderedcorp/coderedcms#327]
I am almost done with creating a website and I came across this cms, so I wonder if I can Integrate it into my wagtail project?

**Comment by eeintech** [comment]
Definitely is possible, I did this myself, you'll have access to the CodeRedCMS pages that you can create content from.

**Comment by safaci2000** [comment]
I did this not long ago.  The main downside is that you'll have to revisit the whole styling of the pages.  Any of my current 'pages' have a certain look that needs to be updated for the CodeRedPages.  Otherwise everything else works, dynamic menus, all the variety of pages that CodeRed Provides etc.

**Comment by vsalvino** [comment]
This is completely possible, and in fact encouraged. Unfortunately we do not have a good guide on this, but here is the general idea:

1. In a new folder, follow the coderedcms quickstart to create a new "boilerplate" project (I'll call it "myproject").
2. Copy the `myproject/website/` folder into the wagtail project. Update the wagtail project's django settings file to include the new `'website'` app in `INSTALLED_APPS`.
3. Review the settings from `myproject/myproject/settings/base.py`, and add the necessary settings to your existing wagtail project's settings. You should also do the same with `myproject/myproject/urls.py`.
4. At this point you can being using coderedcms within the new `website` django app. Follow the example in `website/models.py` to add various coderedcms classes, mixins, and blocks to your other wagtail page models as needed.
5. Don't forget to run `python manage.py makemigrations website` and `python manage.py migrate` when doing model changes.

**Comment by OBKF** [comment]
Thank you all for the help, 

My website isn't that dependent on Wagtail, I just need to add a way for editors to post updates and blog pages so their audience can have updates, so no worry about styling and such.

I will give it a try and ask questions if I get stuck, and thank you all again.

**Comment by OBKF** [comment]
@vsalvino Thank, I ended up porting the whole thing to a new Coderedcms project since I needed to change a lot of things (bad design by me :p ).

So do I close this or leave it open for future references?

**Comment by msantoshk** [comment]
@OBKF you are right. one day I thought about why I selected coderedcms unnecessaryly. whereas I can do this same using wagtail but later got the clarity that even selecting coderedcms I can do the same customization as a wagtail. the only issue which always faces is wagtail updates are very frequent and coderedcms not integrating an update as soon as a new release of wagtail comes.
